spec/symengine_spec.rb

```diff
@@ -18,5 +18,12 @@
     it { is_expected.to be_a SymEngine::Rational }
     its(:to_s) { is_expected.to eq '1/3' }
   end
+  
+  describe 'parse' do
+    subject { SymEngine::parse('123 + 321') }
+    
+    it { is_expected.to be_a SymEngine::Integer }
+    it { is_expected.to eq 444 }
```

Comment:
@zverok I did think about that. In C++ code it will throw errors for unparseable code. Checking that may almost be equivalent to doing the parsing. There are 4 errors,
1. Expected token!
2. Invalid symbol or number!
3. Mismatching parantheses!
4. Operator inconsistency!

Catching the exceptions from the C++ code and throwing ruby exceptions will be covered in the weeks 9 and 10. So this issue will be solved then.

As of now, I wanted the tests to merely confirm that the Ruby wrapper is communicating successfully with the cwrapper. Actually the same idea was behind the matrix wrapper tests.